src/commands/sync.ts

```diff
@@ -354,7 +369,7 @@ export default class Sync extends BaseCommand {
       process.on(signal, () => {
         if (this.status != SyncStatus.RUNNING) return;
 
-        this.log(" Stopping... (press Ctrl+C again to force)");
+        this.log(chalk` Stopping... {gray (press Ctrl+C again to force)}`);
```

Comment:
Checked and `chalk` automatically supports detecting color/no-color support which is nice